src/types/Command.ts

```diff
@@ -73,4 +74,5 @@ export type Command =
   | FeedbackModalCommand
   | ThumbsCommand
   | ManageOrg2Fa
-  | CreateServiceAcc;
+  | CreateServiceAcc
+  | ResponseCommand;
```

Comment:
Lets not add the raw command in here - we should transform the `Responsecommand` to one of the others (if it is known)